src/mumble/AudioConfigDialog.cpp

```diff
@@ -384,15 +386,25 @@ void AudioInputDialog::updateBitrate() {
 	qsQuality->setMinimum(8000);
 }
 
-void AudioInputDialog::on_qcbPushClick_clicked(bool b) {
-	qpbPushClickBrowseOn->setEnabled(b);
-	qpbPushClickBrowseOff->setEnabled(b);
-	qpbPushClickPreview->setEnabled(b);
-	qpbPushClickReset->setEnabled(b);
-	qlePushClickPathOn->setEnabled(b);
-	qlePushClickPathOff->setEnabled(b);
-	qlPushClickOn->setEnabled(b);
-	qlPushClickOff->setEnabled(b);
+void AudioInputDialog::on_qcbEnableCuePTT_clicked() {
+	setPushClickEnabled();
+}
+
+void AudioInputDialog::on_qcbEnableCueVAD_clicked() {
+	setPushClickEnabled();
+}
+
+void AudioInputDialog::setPushClickEnabled() {
```

Comment:
Maybe a better name (that doesn't suggest it definitely enables something) would be something like `updatePushClickStatus` (or *State).

And maybe we can find a better name than "pushClick" while changing this. Imo this is rather undescriptive (not your fault - I do realize this is what this used to be called before).
Maybe just use "AudioCue" instead?